dapytains/app/app.py

```diff
@@ -280,7 +290,7 @@ def document_route():
         db.drop_all()
         db.create_all()
 
-        catalog, _ = parse(os.getenv("DTSCATALOG", f"{basedir}/../../tests/catalog/example-collection.xml"))
+        catalog, _ = parse(os.getenv("DTSCATALOG", "tests/catalog/example-collection.xml"))
         store_catalog(catalog)
 
-    app.run()
+    app.run(debug=True, host=os.getenv("SERVER_HOST", "0.0.0.0"), port=os.getenv("SERVER_PORT", 5000))
```

Comment:
note: if changes are accepted, a PR will follow to install waitress for production environment